plugins/modules/podman_container.py

```diff
@@ -1766,11 +1767,13 @@ def stopped(self):
 
     def get_info(self):
         """Inspect container and gather info about it."""
+        # pylint: disable=unused-variable
         rc, out, err = self.module.run_command(
```

Comment:
Instead of adding ignore comments, it's better to actually fix problems.
In this case it's as easy as adding an underscore to unused vars. Like `err` -> `_err`.

Reply:
Well, it won't work "out of box", you need to pass `--dummy-variables-rgx=^_.*` to pylint for that.

Reply:
Hm... Well, it's set up in a typical pylint default config. I guess I forgot that Ansible's preset may be different.